<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>authPage</class>
 <widget class="QWidget" name="authPage">
  <property name="windowModality">
   <enum>Qt::WindowModality::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>816</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
     <property name="spacing">
      <number>50</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="labelLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>75</number>
       </property>
       <property name="bottomMargin">
        <number>50</number>
       </property>
       <item alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignVCenter">
        <widget class="QLabel" name="logoLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>100</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>General Sans</family>
           <pointsize>62</pointsize>
           <italic>false</italic>
           <fontweight>DemiBold</fontweight>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: #4BA690;
font-size: 62pt;
font-weight: 600;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
         <property name="text">
          <string>Synapso</string>
         </property>
         <property name="textFormat">
          <enum>Qt::TextFormat::AutoText</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="registerPage">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="registerFormLayout" stretch="0,0">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <layout class="QVBoxLayout" name="registerInputLayout">
             <property name="spacing">
              <number>20</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>50</number>
             </property>
             <item alignment="Qt::AlignmentFlag::AlignHCenter">
              <widget class="QLineEdit" name="registerUsername">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>600</width>
                 <height>45</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>800</width>
                 <height>45</height>
                </size>
               </property>
               <property name="layoutDirection">
                <enum>Qt::LayoutDirection::LeftToRight</enum>
               </property>
               <property name="autoFillBackground">
                <bool>false</bool>
               </property>
               <property name="styleSheet">
                <string notr="true">QLineEdit {
    border: 1px solid #3c8573;
    border-radius: 16px;
    height: 25px;
    padding: 10px 12px;
    background-color: #151515;
    color: #fafafa;
	font: 10pt;
}

QLineEdit:hover {
    border: 1px solid #93cabc;
	color: #3c8573;
}

QLineEdit:focus {
    border: 1px solid #6fb8a6;
    background-color: #191919;
}
</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="alignment">
                <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item alignment="Qt::AlignmentFlag::AlignHCenter">
              <widget class="QLineEdit" name="registerEmail">
               <property name="minimumSize">
                <size>
                 <width>600</width>
                 <height>45</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>800</width>
                 <height>45</height>
                </size>
               </property>
               <property name="layoutDirection">
                <enum>Qt::LayoutDirection::LeftToRight</enum>
               </property>
               <property name="autoFillBackground">
                <bool>false</bool>
               </property>
               <property name="styleSheet">
                <string notr="true">QLineEdit {
    border: 1px solid #3c8573;
    border-radius: 16px;
    height: 25px;
    padding: 10px 12px;
    background-color: #151515;
    color: #fafafa;
	font: 10pt;
}

QLineEdit:hover {
    border: 1px solid #93cabc;
	color: #3c8573;
}

QLineEdit:focus {
    border: 1px solid #6fb8a6;
    background-color: #191919;
}
</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item alignment="Qt::AlignmentFlag::AlignHCenter">
              <widget class="QLineEdit" name="registerPassword">
               <property name="minimumSize">
                <size>
                 <width>600</width>
                 <height>45</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>800</width>
                 <height>45</height>
                </size>
               </property>
               <property name="mouseTracking">
                <bool>true</bool>
               </property>
               <property name="layoutDirection">
                <enum>Qt::LayoutDirection::LeftToRight</enum>
               </property>
               <property name="autoFillBackground">
                <bool>false</bool>
               </property>
               <property name="styleSheet">
                <string notr="true">QLineEdit {
    border: 1px solid #3c8573;
    border-radius: 16px;
    height: 25px;
    padding: 10px 12px;
    background-color: #151515;
    color: #fafafa;
	font: 10pt;
}

QLineEdit:hover {
    border: 1px solid #93cabc;
	color: #3c8573;
}

QLineEdit:focus {
    border: 1px solid #6fb8a6;
    background-color: #191919;
}
</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="alignment">
                <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item alignment="Qt::AlignmentFlag::AlignHCenter">
              <widget class="QLineEdit" name="registerRePassword">
               <property name="minimumSize">
                <size>
                 <width>600</width>
                 <height>45</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>800</width>
                 <height>45</height>
                </size>
               </property>
               <property name="layoutDirection">
                <enum>Qt::LayoutDirection::LeftToRight</enum>
               </property>
               <property name="autoFillBackground">
                <bool>false</bool>
               </property>
               <property name="styleSheet">
                <string notr="true">QLineEdit {
    border: 1px solid #3c8573;
    border-radius: 16px;
    height: 25px;
    padding: 10px 12px;
    background-color: #151515;
    color: #fafafa;
	font: 10pt;
}

QLineEdit:hover {
    border: 1px solid #93cabc;
	color: #3c8573;
}

QLineEdit:focus {
    border: 1px solid #6fb8a6;
    background-color: #191919;
}
</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="alignment">
                <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="registerSumbitLayout" stretch="0,0,0">
             <property name="spacing">
              <number>25</number>
             </property>
             <property name="sizeConstraint">
              <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="registerStatus">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>40</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 11pt;
font-weight: 400;</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="alignment">
                <set>Qt::AlignmentFlag::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item alignment="Qt::AlignmentFlag::AlignHCenter">
              <widget class="QPushButton" name="btnRegister">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>400</width>
                 <height>70</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>800</width>
                 <height>80</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>General Sans</family>
                 <pointsize>11</pointsize>
                 <italic>false</italic>
                 <bold>false</bold>
                </font>
               </property>
               <property name="cursor">
                <cursorShape>ArrowCursor</cursorShape>
               </property>
               <property name="mouseTracking">
                <bool>true</bool>
               </property>
               <property name="accessibleDescription">
                <string/>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton#btnRegister {
    border: 1px solid #3c8573;
    border-radius: 16px;
    height: 35px;
    padding: 6px 12px;
    background-color: #151515;
    color: #fafafa;
    font: 11pt;
	font-weight: 400;
}

QPushButton#btnRegister:hover {
    border: 1px solid #93cabc;
    background-color: #181818;
}

QPushButton#btnRegister:pressed {
    background-color: #0f0f0f;
    border: 1px solid #6fb8a6;
}
</string>
               </property>
               <property name="text">
                <string>Registrovať</string>
               </property>
               <property name="checkable">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignVCenter">
              <widget class="QPushButton" name="btnSwitchToLogin">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>50</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>General Sans</family>
                 <pointsize>11</pointsize>
                 <italic>false</italic>
                 <bold>false</bold>
                </font>
               </property>
               <property name="layoutDirection">
                <enum>Qt::LayoutDirection::LeftToRight</enum>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton#btnSwitchToLogin {
    border: none;
    color: #fafafa;
    font: 11pt;
    font-weight: 400;
    background-color: transparent;
}

QPushButton#btnSwitchToLogin:hover {
    color: #3e8e7a;
}</string>
               </property>
               <property name="text">
                <string>Chcem sa prihlásiť.</string>
               </property>
               <property name="flat">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="loginPage">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="loginFormLayout">
           <item>
            <layout class="QVBoxLayout" name="loginInputLayout">
             <property name="spacing">
              <number>20</number>
             </property>
             <property name="topMargin">
              <number>50</number>
             </property>
             <property name="bottomMargin">
              <number>50</number>
             </property>
             <item alignment="Qt::AlignmentFlag::AlignHCenter">
              <widget class="QLineEdit" name="loginUsername">
               <property name="minimumSize">
                <size>
                 <width>600</width>
                 <height>45</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>800</width>
                 <height>45</height>
                </size>
               </property>
               <property name="layoutDirection">
                <enum>Qt::LayoutDirection::LeftToRight</enum>
               </property>
               <property name="autoFillBackground">
                <bool>false</bool>
               </property>
               <property name="styleSheet">
                <string notr="true">QLineEdit {
    border: 1px solid #3c8573;
    border-radius: 16px;
    height: 25px;
    padding: 10px 12px;
    background-color: #151515;
    color: #fafafa;
	font: 10pt;
}

QLineEdit:hover {
    border: 1px solid #93cabc;
	color: #3c8573;
}

QLineEdit:focus {
    border: 1px solid #6fb8a6;
    background-color: #191919;
}
</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item alignment="Qt::AlignmentFlag::AlignHCenter">
              <widget class="QLineEdit" name="loginPassword">
               <property name="minimumSize">
                <size>
                 <width>600</width>
                 <height>45</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>800</width>
                 <height>45</height>
                </size>
               </property>
               <property name="layoutDirection">
                <enum>Qt::LayoutDirection::LeftToRight</enum>
               </property>
               <property name="autoFillBackground">
                <bool>false</bool>
               </property>
               <property name="styleSheet">
                <string notr="true">QLineEdit {
    border: 1px solid #3c8573;
    border-radius: 16px;
    height: 25px;
    padding: 10px 12px;
    background-color: #151515;
    color: #fafafa;
	font: 10pt;
}

QLineEdit:hover {
    border: 1px solid #93cabc;
	color: #3c8573;
}

QLineEdit:focus {
    border: 1px solid #6fb8a6;
    background-color: #191919;
}
</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="alignment">
                <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="loginLayout" stretch="0,0,0">
             <property name="spacing">
              <number>25</number>
             </property>
             <property name="sizeConstraint">
              <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="loginStatus">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>40</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 11pt;
font-weight: 400;</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="alignment">
                <set>Qt::AlignmentFlag::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item alignment="Qt::AlignmentFlag::AlignHCenter">
              <widget class="QPushButton" name="btnLogin">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>400</width>
                 <height>70</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>800</width>
                 <height>80</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>General Sans</family>
                 <pointsize>11</pointsize>
                 <italic>false</italic>
                 <bold>false</bold>
                </font>
               </property>
               <property name="cursor">
                <cursorShape>ArrowCursor</cursorShape>
               </property>
               <property name="mouseTracking">
                <bool>true</bool>
               </property>
               <property name="accessibleDescription">
                <string/>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton#btnLogin {
    border: 1px solid #3c8573;
    border-radius: 16px;
    height: 35px;
    padding: 6px 12px;
    background-color: #151515;
    color: #fafafa;
    font: 11pt;
	font-weight: 400;
}

QPushButton#btnLogin:hover {
    border: 1px solid #93cabc;
    background-color: #181818;
}

QPushButton#btnLogin:pressed {
    background-color: #0f0f0f;
    border: 1px solid #6fb8a6;
}
</string>
               </property>
               <property name="text">
                <string>Prihlásiť</string>
               </property>
               <property name="checkable">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignVCenter">
              <widget class="QPushButton" name="btnSwitchToRegister">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>50</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>General Sans</family>
                 <pointsize>11</pointsize>
                 <italic>false</italic>
                 <bold>false</bold>
                </font>
               </property>
               <property name="layoutDirection">
                <enum>Qt::LayoutDirection::LeftToRight</enum>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton#btnSwitchToRegister {
    border: none;
    color: #fafafa;
    font: 11pt;
    font-weight: 400;
    background-color: transparent;
}

QPushButton#btnSwitchToRegister:hover {
    color: #3e8e7a;
}</string>
               </property>
               <property name="text">
                <string>Chcem sa zaregistrovať.</string>
               </property>
               <property name="flat">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
